docs/proto/provider.md

```
@@ -609,6 +610,24 @@
  | `storage` | [StorageParams](#akash.manifest.v2beta3.StorageParams) | repeated |  |
  | `credentials` | [ImageCredentials](#akash.manifest.v2beta3.ImageCredentials) |  |  |
+ | `permissions` | [ServicePermissions](#akash.manifest.v2beta3.ServicePermissions) |  |  |
+ 
+ 
+
+ 
+
+ 
+ <a name="akash.manifest.v2beta3.ServicePermissions"></a>
+
+ ### ServicePermissions
```

markdownlint-cli2 (0.21.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)